modules/nw-metallb-frr-k8s-configuration-crd.adoc

```diff
@@ -0,0 +1,407 @@
+You must include the prefixes associated with a local preference in the list of prefixes that you want to advertise.
+
+|`spec.bgp.routers.neighbors.toAdvertise.withCommunity.prefixes`
+|`string array`
+|Specifies the prefixes associated with the community.
+
+|`spec.bgp.routers.neighbors.toAdvertise.withCommunity.community`
+|`string`
+|Specifies the community associated with the prefixes.
+
+|`spec.bgp.routers.neighbors.toReceive`
+|`array`
+|Specifies the prefixes to receive from a neighbor.
+
+|`spec.bgp.routers.neighbors.toReceive.allowed`
+|`array`
+|Specifies the information that you want to receive from a neighbor.
+
+|`spec.bgp.routers.neighbors.toReceive.allowed.prefixes`
+|`array`
+|Specifies the prefixes allowed from a neighbor.
+
+|`spec.bgp.routers.neighbors.toReceive.allowed.mode`
+|`string`
+|Specifies the mode to use when handling the prefixes. 
+When set to `filtered`, only the prefixes in the `prefixes` list are allowed. 
+When set to `all`, all the prefixes configured on the router are allowed.
+
+|`spec.bgp.routers.neighbors.disableMP`
+|`boolean`
+|Disables MP BGP to prevent it from separating IPv4 and IPv6 route exchanges into distinct BGP sessions.
+
+|`spec.bgp.routers.prefixes`
+|`string array`
+|Specifies all prefixes to advertise from this router instance.
+
+|`spec.bgp.bfdProfiles`
+|`array`
+|Specifies the list of bfd profiles to use when configuring the neighbors.
+
+|`spec.bgp.bfdProfiles.name`
+|`string`
+|The name of the BFD Profile to be referenced in other parts of the configuration.
+
+|`spec.bgp.bfdProfiles.receiveInterval`
+|`integer`
+|Specifies the minimum interval at which this system can receive control packets, in milliseconds. 
+Defaults to `300ms`.
+
+|`spec.bgp.bfdProfiles.transmitInterval`
+|`integer`
+|Specifies the minimum transmission interval, excluding jitter, that this system wants to use to send BFD control packets, in milliseconds. 
+Defaults to `300ms`.
+
+|`spec.bgp.bfdProfiles.detectMultiplier`
+|`integer`
+|Configures the detection multiplier to determine packet loss. 
+To determine the connection loss-detection timer, multiply the remote transmission interval by this value.
+
+|`spec.bgp.bfdProfiles.echoInterval`
+|`integer`
+|Configures the minimal echo receive transmission-interval that this system can handle, in milliseconds. 
+Defaults to `50ms`.
+
+|`spec.bgp.bfdProfiles.echoMode`
+|`boolean`
+|Enables or disables the echo transmission mode. 
+This mode is disabled by default, and not supported on multihop setups.
+
+|`spec.bgp.bfdProfiles.passiveMode`
+|`boolean`
+|Mark session as passive. A passive session does not attempt to start the connection and waits for control packets from peers before it begins replying.
+
+|`spec.bgp.bfdProfiles.MinimumTtl`
+|`integer`
+|For multihop sessions only. 
+Configures the minimum expected TTL for an incoming BFD control packet.
+
+|`spec.nodeSelector`
+|Limits the nodes that attempt to apply this configuration. 
```

Comment:
I'm jw if this is supposed to be a Type? It seems like its supposed to be a Description but its in the Type column.